winbuild.bat

```diff
@@ -0,0 +1,7 @@
+rmdir /S /Q x64
+call phpize
+call configure --enable-scoutapm --enable-debug --with-php-build="C:\php-sdk\phpdev\vs16\x64\deps" --with-prefix="C:\php\"
+call sed -i 's/-d extension=/-d zend_extension=/g' Makefile
```

Comment:
The need to do that is somewhat unfortunate; https://github.com/php/php-src/pull/8787 would make that superfluous, when using `--enable-test-ini`.